rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java

```diff
@@ -110,18 +107,16 @@ private void flushAll() {
         totalTime = System.nanoTime() - saveTime;
 
         if (logger.isTraceEnabled()) {
-            logger.trace("receiptstore flush: [{}]seconds", FormatUtils.formatNanosecondsToSeconds(totalTime));
+            logger.trace("receiptstore flush: [{}]nano", totalTime);
```

Comment:
Why in nano?

Reply:
Apparently, PAPYRUS-2.2.0-rc does not have the formatter to seconds